src/common/types/cpuArchitecture.ts

```diff
@@ -1,25 +1,69 @@
+import { SupportLevels } from '../api';
+
 export enum CpuArchitecture {
   x86 = 'x86_64',
   ARM = 'arm64',
   MULTI = 'multi',
   // This value refers to use whichever CPU architecture was selected for the cluster
   // It's not a value that can be returned from the Backend.
   USE_DAY1_ARCHITECTURE = 'cluster-day1-cpu-architecture',
+  ppc64le = 'ppc64le',
+  s390x = 's390x',
 }
 export enum OcmCpuArchitecture {
   x86 = 'amd64',
   ARM = 'arm64',
   MULTI = 'multi',
+  ppc64le = 'ppc64le',
+  s390x = 's390x',
+}
+export enum NewCpuArchitecture {
```

Comment:
Having 3 enums for CpuArchitecture can be problematic.

I suggest renaming this to `CpuArchitectureByFeatureId` and maybe converting it to a map of
FeatureId(ARM64_ARCHITECTURE, PPC64LE_ARCHITECTURE etc) --> SupportedCpuArchitecture